DSharpPlus/Entities/DiscordTemplateGuild.cs

```diff
@@ -0,0 +1,121 @@
+﻿using System;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using DSharpPlus.Net.Serialization;
+using Newtonsoft.Json;
+
+namespace DSharpPlus.Entities
+{
+    public class DiscordTemplateGuild : SnowflakeObject
```

Comment:
This class is unnecessary, just use a normal `DiscordGuild` for the partial classes.